src/Domains/Subscription/Plans/Repositories/PlanRepository.php

```diff
@@ -25,7 +25,7 @@ public static function getModel(): Model
         return new Plan();
     }
 
-    public static function getByIdWithApp(int $id, ?AppInterface $app = null): Model
+    public static function getByIdWithApp(string $id, ?AppInterface $app = null): Model
```

Comment:
@Keysie27 you can take both int|string or will it always be string?

Reply:
modified, it has to be int